### ISSUE TITLE
Svelte 5 - "illegal variable name" when using stores

### ISSUE THREAD

**Issue by rChaoz** [sveltejs/eslint-plugin-svelte#746]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.57.0

### What version of `eslint-plugin-svelte` are you using?

2.38.0

### What did you do?

```svelte
<script>
    import { page } from "$app/stores"
    const something = $page
</script>
```

### What did you expect to happen?

No errors.

### What actually happened?

```
  6:34  error  `$page` is an illegal variable name. To reference a global variable called `$page`, use `globalThis.$page`(global_reference_invalid)  svelte/valid-compile
```

### Link to **GitHub Repo** with Minimal Reproducible Example

-

### Additional comments

This happens for all stores in `.svelte` files. I haven't checked but the same issue might happen in `.svelte.[ts|js]` files.

**Comment by ota-meshi** [comment]
Duplicate of #652
